public/style/components/navigation.css

```diff
@@ -57,11 +56,11 @@ nav ul li.donate a:hover
     background-color: var(--color-mystlue);
 }
 
-@media screen and (max-width: 860px)
-{
-    nav ul li.donate
-    {
-        position: relative;
-        top: 5px;
+@media only screen and (max-width: 600px) {
+    nav {
+        font-size: 17px;
     }
-}
+    nav ul {
+        padding: 0.3rem 0.4rem;
+    }
+}
```

Comment:
should follow the same style (brace on new line)

Reply:
ok. Will be fixing :+1: